src/types/activity.ts

```diff
@@ -69,6 +69,10 @@ export function getStatusText(status: ParticipantStatus) {
   return STATUS_TEXT[status];
 }
 
+export function getOrganizationName(activity: Activity, organizationId: string) {
+  return activity.organizations[organizationId].rosterName ?? activity.organizations[organizationId].title;
```

Comment:
I noticed you have this same logic, but inline rather than as a selector, in your other PR. Can you make sure this selector is used there as well? However you want to do that.

Reply:
Yeah, I can update the other PR after this is merged. The dependency doesn't exist yet, obviously.